Application/Api/Billing/Bookkeeping/ApiBasketRepaymentAddPerson.php

```diff
@@ -0,0 +1,903 @@
+use SPHERE\Common\Frontend\Icon\Repository\Exclamation;
+use SPHERE\Common\Frontend\Icon\Repository\Info as InfoIcon;
+use SPHERE\Common\Frontend\Icon\Repository\Ok;
+use SPHERE\Common\Frontend\Icon\Repository\Save;
+use SPHERE\Common\Frontend\Icon\Repository\Search;
+use SPHERE\Common\Frontend\Icon\Repository\Warning as WarningIcon;
+use SPHERE\Common\Frontend\Layout\Repository\Listing;
+use SPHERE\Common\Frontend\Layout\Repository\Panel;
+use SPHERE\Common\Frontend\Layout\Repository\Well;
+use SPHERE\Common\Frontend\Layout\Structure\Layout;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutColumn;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutGroup;
+use SPHERE\Common\Frontend\Layout\Structure\LayoutRow;
+use SPHERE\Common\Frontend\Link\Repository\Danger as DangerLink;
+use SPHERE\Common\Frontend\Link\Repository\Primary;
+use SPHERE\Common\Frontend\Message\Repository\Danger;
+use SPHERE\Common\Frontend\Message\Repository\Success;
+use SPHERE\Common\Frontend\Message\Repository\Warning;
+use SPHERE\Common\Frontend\Table\Structure\TableData;
+use SPHERE\Common\Frontend\Text\Repository\Bold;
+use SPHERE\Common\Frontend\Text\Repository\ToolTip;
+use SPHERE\Common\Frontend\Text\Repository\Warning as WarningText;
+use SPHERE\System\Extension\Extension;
+
+/**
+ * Class ApiBasketRepaymentAddPerson
+ * @package SPHERE\Application\Api\Billing\Bookkeeping
+ */
+class ApiBasketRepaymentAddPerson extends Extension implements IApiInterface
+{
+
+    // registered method
+    use ApiTrait;
+
+    public function exportApi($Method = '')
+    {
+        $Dispatcher = new Dispatcher(__CLASS__);
+        // AddDebtorSelection
+        $Dispatcher->registerMethod('showSearchModal');
+        $Dispatcher->registerMethod('showSearch');
+        $Dispatcher->registerMethod('addDebtorSelection');
+
+        // reload Table
+        $Dispatcher->registerMethod('getTableLayout');
+
+        // Item Price
+        $Dispatcher->registerMethod('showItemPrice');
+        $Dispatcher->registerMethod('checkItemPrice');
+        $Dispatcher->registerMethod('changeItemPrice');
+
+        // BankAccount
+        $Dispatcher->registerMethod('showEditBankAccount');
+        $Dispatcher->registerMethod('saveEditBankAccount');
+
+        // remove Entry
+        $Dispatcher->registerMethod('showDeleteBasketSelection');
+        $Dispatcher->registerMethod('deleteDebtorSelection');
+
+
+        return $Dispatcher->callMethod($Method);
+    }
+
+    /**
+     * @param string $Header
+     * @param string $Identifier
+     *
+     * @return ModalReceiver
+     */
+    public static function receiverModal($Header = '', $Identifier = '')
+    {
+
+        return (new ModalReceiver($Header, new Close()))->setIdentifier('Modal'.$Identifier);
+    }
+
+    /**
+     * @param string $Content
+     * @param string $Identifier
+     *
+     * @return BlockReceiver
+     */
+    public static function receiverSearch($Content = '', $Identifier = '')
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Content" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)